eclair-node-gui/src/main/scala/fr/acinq/eclair/gui/GUIUpdater.scala

```diff
@@ -101,11 +101,11 @@ class GUIUpdater(mainController: MainController) extends Actor with ActorLogging
       val channelPaneController = m(channel)
       runInGuiThread(() => channelPaneController.channelId.setText(channelId.toHex))
 
-    case ChannelStateChanged(channel, _, _, _, currentState, currentData) if m.contains(channel) =>
+    case ChannelStateChanged(channel, _, _, _, _, currentState, Some(commitments)) if m.contains(channel) =>
```

Comment:
There's a small regression here:
```suggestion
    case ChannelStateChanged(channel, _, _, _, _, currentState, commitments_opt) if m.contains(channel) =>
```